src/main/java/gov/usgs/traveltimeservice/TravelTimeControllerClient.java

```diff
@@ -0,0 +1,20 @@
+package gov.usgs.traveltimeservice;
+
+import gov.usgs.processingformats.TravelTimeException;
+import gov.usgs.processingformats.TravelTimePlotRequest;
+import gov.usgs.processingformats.TravelTimeRequest;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.client.annotation.Client;
+import javax.validation.constraints.NotBlank;
+
+@Client("${traveltime.controller.path:/traveltime}")
+public interface TravelTimeControllerClient {
+
+  @Post("/traveltime")
+  public TravelTimeRequest getTravelTime(@NotBlank TravelTimeRequest request)
+      throws TravelTimeException;
+
+  @Post("/traveltimeplot")
+  public TravelTimePlotRequest getTravelTime(@NotBlank TravelTimePlotRequest request)
```

Comment:
should this be getTravelTimePlot? both /traveltime and /traveltimeplot make the same function call but I do see there is a different type of request object, just clarifying that is correct? if so maybe worth adding in a comment because reading the code it is not so clear...